test/stripe/issuing/authorization_test.rb

```diff
@@ -32,18 +32,40 @@ class AuthorizationTest < Test::Unit::TestCase
         assert authorization.is_a?(Stripe::Issuing::Authorization)
       end
 
-      should "be approveable" do
-        authorization = Stripe::Issuing::Authorization.retrieve("iauth_123")
-        authorization.approve
-        assert_requested :post, "#{Stripe.api_base}/v1/issuing/authorizations/iauth_123/approve"
-        assert authorization.is_a?(Stripe::Issuing::Authorization)
+      context "#approve" do
+        should "approve an authorization" do
```

Comment:
Also, bit of a nitpick, but when testing an instance method we can replace "a/an" by "the" (e.g. "approve the authorization") since we know which authorization we're talking about (the one represented by the instance).